packages/api-generator/src/locale/en/v-overflow-btn.json

```diff
@@ -2,6 +2,7 @@
   "props": {
     "editable": "Creates an editable button",
     "overflow": "Creates an overflow button",
-    "segmented": "Creates a segmented button"
+    "segmented": "Creates a segmented button",
+    "persistentPlaceholder": "Forces label to always be visible"
```

Comment:
You can omit this, Inherited props use the description from the source component.

Reply:
I've changed it a little, because in source component it says 'Forces placeholder to always be visible', but in case of VOverflowBtn it's not placeholder but label

Reply:
Fair enough